class/practice_3.2/test.cpp

```diff
@@ -0,0 +1,100 @@
+#include <print>
+#include <Helpers.hpp>
+#include <UnitTests.hpp>
+
+int main() {
+    UnitTests testSuite;
+
+    testSuite.addTest("Dummy_test1",
+    [](){
+        // Build:
+        double value = 0.4;
+    
+        // Operate:
+        auto result = dummyFunc(value);
+
+        // Check:
+        ASSERT_EQ(result, 0.4)
+    });
+    testSuite.addTest("Dummy_test2",
+    [](){
+        // Build:
+        double value = 0.0;
+    
+        // Operate:
+        auto result = dummyFunc(value);
+
+        // Check:
+        ASSERT_EQ(result, 0.0)
+    });
+    testSuite.addTest("Triangle_test1", 
+    [](){
+        Triangle tri(3.0, 4.0, 5.0);
+        double result = tri.calculateArea();
+        ASSERT_EQ(result, 6.0);
+        return true;
```

Comment:
no need to return. our mini-framework expects void return function.
In unit tests we have assert functions to handle status of test execution